setup.py

```diff
@@ -26,7 +21,7 @@ def get_requirements():
             "Programming Language :: Python :: 2.7",
             "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
       ],
-      install_requires=get_requirements(),
+      install_requires=["koji", "rpm", "requests", "PyYAML", "simplejson"],
```

Comment:
Could we please avoid making this change?  Having the deps in a separate requirements.txt is useful in a few cases, it allows other tools (like github) to understand the project's dependencies and it makes it easier to patch them during RPM builds when that occasionally becomes needed.

You will probably find that adding "include requirements.txt" into MANIFEST.in would also fix whatever problem this was meant to resolve.  As I understand it, the issue is that current versions of setuptools include requirements.txt in the default list of files packaged for a project, but older versions don't, and Py 2.6 tests are using those older versions.

Reply:
Looks like MANIFEST.in did fix the issue.